### PR TITLE
[Feature] Show disclaimer if report from manifest

### PR DESCRIPTION
**Pull request by wcchang1115**
**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
feature

**What this PR does / why we need it**:
- need disclaimer to hint user when reports are generated from manifest file

**Which issue(s) this PR fixes**:
sc-32348

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

NONE

### CODE REVIEW

**Review by popcornylu — CHANGES_REQUESTED** [review]
1. Make the `schema_type=''`, so that we can show `-` if the schema_type is empty string (not fetched from data warehouse)
2. `store.ts`: use the existing `rawData` is enough. 
3. nit: Would be better to show the data source is skipped in the first icon in the banner.

File: static_report/src/utils/store.ts [hunk 1]
```diff
@@ -100,6 +101,10 @@ export interface ReportState {
   reportTitle?: string;
   reportTime?: string;
   reportDisplayTime?: string;
+  reportDataSource?: {
```
[the remark this answers is not in the document]
Comment (wcchang1115) [reply static_report/src/utils/store.ts]: Hi pop,
Thank you for the review.
(1) and (2) have been fixed, and for (3) we still keep the new icon but the information is based on the target's POV.

**Review by popcornylu — APPROVED** [review]

**Comment by popcornylu** [comment]
LGTM

**Review by popcornylu — APPROVED** [review]
LGTM

**Pull request merged**